chsdi/templates/htmlpopup/mittlere_abfluesse.mako

```diff
@@ -0,0 +1,153 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<% c['stable_id'] = True %>
+<tr><td
+class="cell-left">${_('ch.bafu.mittlere-abfluesse.mqn_jahr')}</td>
+  % if c['attributes']['mqn_jahr'] != None:
+       <td>${round(c['attributes']['mqn_jahr'],2)}</td></tr>
+  % else:
+       <td>-</td></tr>
```

Comment:
Here's an example of how a round(None) should be avoided (see my comments above where it's not really taken care of properly).